modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -234,25 +406,78 @@ public void testGetObjectEntry() throws Exception {
 
 		Assert.assertEquals(
 			title, MapUtil.getString(objectEntry.getProperties(), "title"));
+	}
 
-		_objectEntryManager.deleteObjectEntry(
-			TestPropsValues.getCompanyId(), _getDTOConverterContext(),
-			objectEntry.getExternalReferenceCode(), _objectDefinition,
-			ObjectDefinitionConstants.SCOPE_COMPANY);
+	private void _assertEquals(
+		List<ObjectEntry> actualObjectEntries,
+		List<ObjectEntry> expectedObjectEntries) {
+
+		Assert.assertEquals(
+			actualObjectEntries.toString(), expectedObjectEntries.size(),
+			actualObjectEntries.size());
+
+		for (int i = 0; i < expectedObjectEntries.size(); i++) {
+			_assertEquals(
+				actualObjectEntries.get(i), expectedObjectEntries.get(i));
+		}
+	}
+
+	private void _assertEquals(
+		ObjectEntry actualObjectEntry, ObjectEntry expectedObjectEntry) {
+
+		Map<String, Object> actualObjectEntryProperties =
+			actualObjectEntry.getProperties();
+
+		Map<String, Object> expectedObjectEntryProperties =
+			expectedObjectEntry.getProperties();
+
+		for (Map.Entry<String, Object> expectedEntry :
+				expectedObjectEntryProperties.entrySet()) {
+
+			Assert.assertEquals(
+				expectedEntry.getKey(), expectedEntry.getValue(),
+				actualObjectEntryProperties.get(expectedEntry.getKey()));
+		}
 	}
 
-	private DTOConverterContext _getDTOConverterContext() throws Exception {
+	private String _buildNotEqualsExpressionFilterString(
+		String fieldName, Object value) {
+
+		return StringBundler.concat(
+			fieldName, " ne ", ObjectEntryTestUtil.getValue(value));
+	}
+
+	private DTOConverterContext _getDTOConverterContext() {
 		return new DefaultDTOConverterContext(
 			false, Collections.emptyMap(), _dtoConverterRegistry, null,
 			LocaleUtil.getDefault(), null, _user);
 	}
 
+	private void _testGetObjectEntries(
```

Comment:
Is there a way to make this method more generic and then reuse it in both test classes? `SalesforceObjectEntryManagerImplTest` and `DefaultObjectEntryManagerImplTest`
Maybe we should create a new class `BaseObjectEntryManagerImplTest`

You can check these examples: BaseNotificationTypeTest and BaseDDMTestCase
It looks like it fits with [Template Method Design Pattern](https://www.digitalocean.com/community/tutorials/template-method-design-pattern-in-java)

Reply:
@guilhermedcamacho  I though about it, but I found some issues about it.
1- **SalesforceObjectEntryManagerImplTest** and **DefaultObjectEntryManagerImplTest** its placed in different modules, so this **BaseObjectEntryManagerImplTest** need to be put where both modules can see it, the unique place that attend this needed its some `-api` module. But I don't know if it is good idea create a test class in a `-api` module